backend/tests/apps/owasp/management/commands/owasp_update_project_health_scores_test.py

```
@@ -81,4 +81,4 @@ def test_handle_successful_update(self):
         )
         assert mock_metric.score == EXPECTED_SCORE
         assert "Updated project health scores successfully." in self.stdout.getvalue()
-        assert "Updating score for project: Test Project" in self.stdout.getvalue()
+        assert "Updating score for project: Test Project" in self.stdout.getvalue()
```

Ruff (0.14.11)

No newline at end of file

Add trailing newline

(W292)